modules/light-clients/08-wasm/types/wasm_test.go

```diff
@@ -61,23 +61,23 @@ func (suite *TypesTestSuite) TestGetCodeHashes() {
 func (suite *TypesTestSuite) TestAddCodeHash() {
 	suite.SetupWasmWithMockVM()
 
-	codeHashes, err := types.GetCodeHashes(suite.chainA.GetContext(), GetSimApp(suite.chainA).AppCodec())
+	codeHashes, err := types.GetAllCodeHashes(suite.chainA.GetContext())
 	suite.Require().NoError(err)
 	// default mock vm contract is stored
-	suite.Require().Len(codeHashes.Hashes, 1)
+	suite.Require().Len(codeHashes, 1)
 
 	codeHash1 := []byte("codehash1")
 	codeHash2 := []byte("codehash2")
-	err = types.AddCodeHash(suite.chainA.GetContext(), GetSimApp(suite.chainA).AppCodec(), codeHash1)
+	err = types.AddCodeHash(suite.chainA.GetContext(), codeHash1)
```

Comment:
what happens if we add the same code hash twice? It should no-op correct? Could we add a test case?

Reply:
added